CHANGELOG.md

```diff
@@ -1,328 +1,711 @@
+* add `AppVeyor` so we can build on Windows (@vweevers)
+* add OSX to travis (@juliangruber)
+* add `prebuild-ci` so we can automatically make prebuilt binaries (@juliangruber)
 
-### 0.6.1 @ Jun 15 2013
+### Changed
+* enable `iterator.seek()` with buffers (@peakji)
 
-  * Fix broken Windows compile, apply port-uv patch to LevelDB's port.h (@rvagg)
+### Removed
+* remove `node@0.10` from travis (@ralphtheninja)
+* remove `node@5` from travis (@ralphtheninja)
 
-### 0.6.0 @ Jun 14 2013
+## [1.5.0] - 2016-09-27
 
-  * Upgrade to LevelDB 1.11.0, some important bugfixes: https://groups.google.com/forum/#!topic/leveldb/vS1JvmGlp4E
+### Changed
+* upgrade to `LevelDB@1.19` (@juliangruber)
+* bump dependencies (@ralphtheninja)
+* upgrade to `nan@2.4.0`, remove `Function::NewInstance()` deprecation warnings (@rvagg)
 
-### 0.5.0 @ May 18 2013
+### Removed
+* remove deleted file from `leveldb.gyp` (@juliangruber)
 
-  * Bumped major version for db.getProperty() addition (should have been done in 0.4.4) (@rvagg)
-  * Disallow batch() operations after a write() (@rvagg)
+## [1.4.6] - 2016-04-29
 
-### 0.4.4 @ May 18 2013
+### Added
+* add node 6 to travis (@ralphtheninja)
 
-  * db.getProperty() implemented, see README for details (@rvagg)
-  * More work on memory management, including late-creation of Persistent handles (@rvagg)
+### Changed
+* upgrade to `nan@2.3.0` to remove deprecated calls (@ralphtheninja)
 
-### 0.4.3 @ May 18 2013
+## [1.4.5] - 2016-04-18
 
-  * Better memory leak fix (@rvagg)
+### Added
+* steal chjjs script for checking memory consumption (@ralphtheninja)
 
-### 0.2.2 @ May 17 2013
+### Changed
+* bump dependencies (@ralphtheninja)
+* gitignore Release folder for Windows (@vweevers)
+* update copyright year 2015 -> 2016 (@ralphtheninja)
+* optimize iterators by avoiding handles (@chjj)
 
-  * BACKPORT memory leak fixes (@rvagg)
+### Removed
+* remove unsupported versions from travis (@ralphtheninja)
 
-### 0.4.2 @ May 17 2013
+### Fixed
+* fix delete calls not using `[]` (@chjj)
+* free start slices on conflicting options (@chjj)
+* dispose of unused slices (@chjj)
+* release iterator snapshots (@chjj)
+* fix iterator leak (@chjj)
+* add handlescopes to fix potential memory leaks (@chjj)
+* fix repair-test for multilang windows (@vweevers)
+* repair error is different on windows (@ralphtheninja)
 
-  * Same memory leak fixes as 0.4.1, properly applied to batch() operations too (@rvagg)
+## [1.4.4] - 2016-01-25
 
-### 0.4.1 @ May 17 2013
+### Changed
+* update dependencies (@ralphtheninja)
+* use `prebuild --install` (@ralphtheninja)
+* `README` fixes (@ralphtheninja)
 
-  * Fix memory leak caused when passing String objects in as keys and values, conversion to Slice created new char[] but wasn't being disposed. Buffers are automatically disposed. (@rvagg, reported by @kylegetson levelup/#140)
+### Fixed
+* fix build build problems on windows https://github.com/Level/leveldown/issues/247 (@obastemur)
```

Comment:
- build build
- I think we can remove the link, seeing as we're not adding links on other items either?